configuration/messageproperties/messages_es.properties

```diff
@@ -3067,6 +3067,10 @@ zl.registration.patient.email.question=Ingresar correo electrónico
 pihcore.referral=Referral
 pihcore.transfer=Transfer
 
+pihcore.hivAlerts.alerts=ALERTS
```

Comment:
Is our generally agreed best practice to make sure that all message codes are in every file, but with english text if not yet translated?  If so, this looks good.